8.1-Chi.sq.tests/lesson.yaml

```diff
@@ -54,7 +54,7 @@
   Output: So, we got 2.25 as our first residual. If you didn't have R, you'd finish up by hand, but fortunately, we can compute the rest automatically!
 
 - Class: cmd_question
-  Output: Try it now using xchisq.test() - xchisq.test(tally(Relapse~Drug, data=CocaineTreatment))
+  Output: Try it now using xchisq.test - xchisq.test(tally(Relapse~Drug, data=CocaineTreatment))
```

Comment:
I changed this, cause I got confused thinking the instructions were telling me to call xchisq.test